pages/pil/categories/views/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react';
+import Snippet from '../../../common/views/containers/snippet';
+import Link from '../../../common/views/containers/link';
+
+const links = ['pil.catAF', 'pil.catE'];
+
+const Index = () => (
+  <Fragment>
+    <header>
+      <h2>&nbsp;</h2>
+      <h1>
+        <Snippet>pages.pil.categories</Snippet>
+      </h1>
+    </header>
+    <div className="grid-row">
```

Comment:
All the class names will need updating now we've merged the HO Design System work.

See: https://design-system.service.gov.uk/get-started/updating-your-code/#layout-and-grid-system-class-names